lib/reporter.ts

```diff
@@ -198,6 +198,14 @@ class ReportPortalReporter extends Reporter {
         level: LEVEL.ERROR,
         message,
       });
+
+      if (this.options.cucumberNestedSteps) {
```

Comment:
is it ok to add the error log in the RP twice?(because earlier it was added to cucumber step). Havent you experienced any issues?

Reply:
Seems the same question as in https://github.com/BorisOsipov/wdio-reportportal-reporter/pull/107#discussion_r383427219

Reply:
> is it ok to add the error log in the RP twice?(because earlier it was added to cucumber step). Havent you experienced any issues?

No issues with having it twice, it just shows as an item in the nested steps.  You'll see the same message at the level of the nested steps, when you expand the step (this is what gets added above for the test item), and when you click stack trace for the scenario.  We viewed it better to have the same message multiple times than to lose the auto-analysis functionality.

> Seems the same question as in [#107 (comment)](https://github.com/BorisOsipov/wdio-reportportal-reporter/pull/107#discussion_r383427219)

Good stuff, thank you!